src/main/java/Control/Scenario/UC_FlatSummary.java

```diff
@@ -115,11 +115,28 @@ public Flat getFlatSummary(@NotNull final Integer flatId) {
         return flat;
     }
 
+    public ControllerUnit getControllerUnitByUid(@NotNull final Integer controllerUnitUid) {
+        // ATTEMPT to eliminate WEBSERVLET only falling asleep of connections
```

Comment:
preloz mi toto pls do slovenciny, falling asleep of connection? co?

Reply:
Čo som ti spomínal, že Connection na serveri vypršal zaspal